ui/dashboards/src/components/Variables/Variable.tsx

```diff
@@ -113,24 +113,11 @@ function ListVariable({ name }: TemplateVariableProps) {
 
   useEffect(() => {
     const firstOption = finalOptions?.[0];
-    const valueIsInOptions = Boolean(
-      finalOptions.find((v) => {
-        if (allowMultiple) {
-          return (value as string[]).includes(v.value);
-        }
-        return value === v.value;
-      })
-    );
 
     // If there is no value but there are options, set the value to the first option.
     if (!value && firstOption) {
       setVariableValue(name, firstOption.value);
     }
-
-    // If there is a value but it's not in the options, select the first value or set to null
-    if (value && !valueIsInOptions && !definition.spec.default_value) {
```

Comment:
Removing this logic for now because this becomes a bit more tricky when you have URL parameters. I think we'll have to think about this more. Should values be allowed if the values are not in the options? My gut is yes, since sometimes options are slow to load / fail.